APCAppCore/APCAppCore/UI/TabBarControllers/Activities/APCActivitiesViewController.m

```diff
@@ -35,28 +35,29 @@
 #import "APCActivitiesViewSection.h"
 #import "APCAppDelegate.h"
 #import "APCBaseTaskViewController.h"
-#import "APCCircularProgressView.h"
-#import "APCConstants.h"
 #import "APCDataMonitor+Bridge.h"
 #import "APCLog.h"
 #import "APCPermissionsManager.h"
 #import "APCScheduler.h"
 #import "APCSpinnerViewController.h"
 #import "APCTask.h"
-#import "APCTaskGroup.h"
 #import "APCTasksReminderManager.h"
-#import "APCUtilities.h"
 #import "NSBundle+Helper.h"
 #import "NSDate+Helper.h"
 #import "UIAlertController+Helper.h"
 #import "UIColor+APCAppearance.h"
-#import "NSDictionary+APCAdditions.h"
+#import "UIFont+APCAppearance.h"
 #import "APCLocalization.h"
 
 
 static CGFloat const kTintedCellHeight             = 65;
 static CGFloat const kTableViewSectionHeaderHeight = 77;
-
+static CGFloat const kActivityHeaderTitleLabelFontSize = 20.f;
+static CGFloat const kActivityHeaderSubTitleLabelFontSize = 12.f;
+static NSString *const kActivityHeaderBackgroundImage = @"Activity_Header_Background";
+static NSString *const kAPCSectionTitleKeepGoing = @"Additional Activities";
+static NSString *const kAPCSectionSubtitleKeepGoing = @"Try one of these extra activities to enhance your study experience.";
+static NSString *const kAPCSectionSubtitleYesterday = @"Below are your incomplete tasks from yesterday. For your reference only.";
```

Comment:
static strings. need to be localized and if you are using as static strings then need to listen for language change and change the language accordingly.